fontforge/sfd.c

```diff
@@ -7465,14 +7465,6 @@ bool SFD_GetFontMetaData( FILE *sfd,
 	geteol(sfd,val);
 	sf->familyname = copy(val);
     }
-    else if ( strmatch(tok,"OS2FamilyName:")==0 )
-    {
-	if (!sf->pfminfo.os2_family_name) sf->pfminfo.os2_family_name = SFDReadUTF7Str(sfd);
-    }
-    else if ( strmatch(tok,"OS2StyleName:")==0 )
-    {
-	if (!sf->pfminfo.os2_style_name) sf->pfminfo.os2_style_name = SFDReadUTF7Str(sfd);
-    }
```

Comment:
In the interest of maximizing backwards compatibility, we ought to process these directives into the new data types with a note that they are deprecated.